<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Grades</class>
 <widget class="QDialog" name="Add_Grades">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Test Numberl</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Score</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Course Id</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addgrades_button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>135</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="Testnum_edit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="score_edit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="courseid_edit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="studentid_edit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Student Id</string>
   </property>
  </widget>
  <zorder>Testnum_edit</zorder>
  <zorder>score_edit</zorder>
  <zorder>courseid_edit</zorder>
  <zorder>studentid_edit</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_4</zorder>
  <zorder>addgrades_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
